Multisig/UI/Safe Management/SwitchSafeView.swift

```diff
@@ -62,6 +61,7 @@ struct SwitchSafeView: View {
             .background(Rectangle().foregroundColor(.white))
             .onTapGesture {
                 self.appSettings.selectedSafe = self.safe.address
+                self.presentationMode.wrappedValue.dismiss()
```

Comment:
Do we need it?

Reply:
Yes in order to dismiss the screen after selecting new safe

Reply:
will it work if we just take it from ```@Environment(\.presentationMode) var presentationMode: Binding<PresentationMode>``` ?